napari_allencell_annotator/view/viewer.py

```diff
@@ -44,4 +58,97 @@ def alert(self, alert_msg: str) -> None:
         show_info(alert_msg)
 
     def get_layers(self) -> List[Layer]:
+        """
+        Returns a list of all layers in the viewer.
+        """
         return list(self.viewer.layers)
+
+    def get_all_points_layers(self) -> List[Points]:
+        """
+        Returns a list of all point layers in the viewer.
+        """
+        return [layer for layer in self.get_layers() if isinstance(layer, Points)]
+
+    @staticmethod
```

Comment:
avoid staticmethods whenever possible

Reply:
make it a class method in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/4b48d1d02ffcaed68cc7909fa35f6ae0d7899928